NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java

```diff
@@ -1,29 +1,7 @@
 package org.bukkit.craftbukkit.util;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Properties;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-import org.bukkit.Bukkit;
-
 public final class Versioning {
     public static String getBukkitVersion() {
-        String result = "Unknown-Version";
-
-        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/dev.cobblesword.nachospigot/api/pom.properties");
-        Properties properties = new Properties();
-
-        if (stream != null) {
-            try {
-                properties.load(stream);
-
-                result = properties.getProperty("version");
-            } catch (IOException ex) {
-                Logger.getLogger(Versioning.class.getName()).log(Level.SEVERE, "Could not get Bukkit version!", ex);
-            }
-        }
-
-        return result;
+        return "1.8.8-R0.2-SNAPSHOT";
```

Comment:
@Lucaskyy Should we finally switch to `1.8.8-R0.1-SNAPSHOT` like upstream?

Reply:
R0.1 is the most used

Reply:
@crafter23456 ^